src/style.css

```
@@ -1 +1,18 @@
 /* styles go here */
+.fieldset {
+  /* margin: auto auto auto; */
+  margin-bottom: 20px;
+}
```

```suggestion
.fieldset {
  margin-bottom: 20px;
}
```
always clear comments before making a pull request